libraries/AP_NavEKF/AP_Nav_Common.h

```diff
@@ -19,6 +19,30 @@
 #include <stdint.h>
 #include <AP_Math/AP_Math.h>
 
+// enumeration corresponding to buts within nav_filter_status union.
+// Only used for documentation purposes.
+enum class NavFilterStatusBit {
```

Comment:
In the past I have put these "documentation only" enums in the function which does the logging so there is very limited scope.